src/main/java/org/plp/implementations/plpisa/PlpFile.java

```diff
@@ -41,8 +43,14 @@ private void validateFilePath(Path filePath) throws IOException {
      * @return true if it is successful to write to file, false if there is an error
      */
     @Override
-    public boolean writeToFile() {
-        return false;
+    public boolean writeToFile() throws IOException{
```

Comment:
Is it possible to mock the IOException during the Buffer Writer in our test cases?
```
@test
void writeToFileException
```

Reply:
Why we want to test it?